src/configure/configure.ts

```diff
@@ -63,16 +78,33 @@ class PipelineConfigurer {
         await this.getAzureDevOpsDetails();
         await this.getSelectedPipeline();
 
-        if (this.inputs.sourceRepository.repositoryProvider === RepositoryProvider.Github) {
-            await this.getGitubConnectionService();
-        }
-
         if (!this.inputs.targetResource.resource) {
             await this.getAzureResourceDetails();
         }
+    }
+
+    private async createPreRequisites(): Promise<void> {
+        if (this.inputs.isNewOrganization) {
+            this.inputs.projectName = generateDevOpsProjectName(this.inputs.sourceRepository.repositoryName);
+            await vscode.window.withProgress(
+                {
+                    location: vscode.ProgressLocation.Notification,
+                    title: Messages.creatingAzureDevOpsOrganization
+                },
+                () => {
+                    return this.azureDevOpsClient.createOrganization(this.inputs.organizationName)
+                        .then(() => {
```

Comment:
What's the end user experience here ? Will this not cause a lot of delay ? as the inputs from the user are taken and this is being done on the backend

Can we do it simultaneously with the inputs to keep user engaged ?

Reply:
As discussed offline, we want to move towards experience where resources created on behalf of user is done only after user confirms the yml file. We already have a backlog item for this. Simultaneous won't help as we don't want to creating any resources while user is selecting other inputs.